### ISSUE TITLE
Updating the app to 0.22 throws exception when migrating

### ISSUE THREAD

**Issue by selaux** [H2CK/oidc#63]
I tried to update the app to `0.2.2` to test the fix for #53, but unfortunately the update runs into an issue with migrations for me:

```
Exception: Database error when running migration 0003Date20220927082100 for app oidc redirectUri is not a valid attribute
```

I use PostgreSQL as the database if that matters.

**Comment by H2CK** [comment]
Sorry for the circumstances. Should be fixed with version 0.2.3.

**Comment by selaux** [comment]
I can confirm it works now. Thanks :heart:
